source/declarations.tex

```diff
@@ -6211,28 +6211,38 @@
 \end{itemize}
 
 \pnum
-The type \tcode{T}$_1$ of an explicitly defaulted special member function \tcode{F}
-is allowed to differ from the type \tcode{T}$_2$ it would have had
-if it were implicitly declared, as follows:
+An explicitly defaulted special member function \tcode{F}$_1$
+with type \tcode{T}$_1$
+is allowed to differ from
+the corresponding special member function \tcode{F}$_2$
+with type \tcode{T}$_2$
+that would have been implicitly declared, as follows:
 \begin{itemize}
 \item
   \tcode{T}$_1$ and \tcode{T}$_2$ may have differing \grammarterm{ref-qualifier}{s};
+\item
+  if \tcode{F}$_2$ has an implicit object parameter of
+  type ``reference to \tcode{C}'',
+  \tcode{F}$_1$ may be an explicit object member function whose
+  explicit object parameter is of type ``reference to \tcode{C}'';
```

Comment:
Suggestion:
https://github.com/cplusplus/draft/issues/5567#issuecomment-1194642110

which makes it clear that T1 and T2 may differ because of this rule for the purposes of the "other than as allowed by the preceding rules" below.

Reply:
Please open a separate issue (or PR) for this; I'd rather not include that change as part of the motions. Thanks!